tests/cpp/utils.cpp

```diff
@@ -824,4 +824,42 @@ std::string macroToString(const MmaMacro macro) {
   return ss.str();
 }
 
+std::pair<TensorView*, TensorView*> getSdpaRngTvs(bool symbolic) {
+  TensorView* philox_seed = nullptr;
+  TensorView* philox_offset = nullptr;
+  DataType dtype = DataType::Int;
+  int64_t philox_ndims = 0;
+  std::vector<int64_t> philox_shape = {};
+
+#if NVF_TORCH_VERSION_NO_LESS(2, 7, 0)
+  dtype = DataType::UInt64;
+  philox_ndims = 1;
+  philox_shape = {2};
+#endif
+
+  if (symbolic) {
+    philox_seed = TensorViewBuilder().ndims(philox_ndims).dtype(dtype).build();
```

Comment:
Why bother testing symbolic tensor views since we know the shape is {2} for >= 2.7 or {} for < 2.7?

Reply:
Yep. I agree. Symbolic vs concrete is not relevant for these. I kept it since some tests use `makeSymbolicTensor` but we can remove it.